example/data/input/credit-balance-mapping.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<data-mapping>
+  <field-mappings>
+    <field-mapping fieldName="Account_ID" sourceExpression="toLong(${source.Account})"/>
```

Comment:
We shouldn't need to convert to long since the schema's mapping should already have done this.

Reply:
done